extensions/amp-base-carousel/1.0/arrow.js

```diff
@@ -71,10 +63,88 @@ export function Arrow(props) {
  * @param {!JsonObject} props
  * @return {PreactDef.Renderable}
  */
-function renderDefaultArrow(props) {
+export function ArrowPrev(props) {
+  const {'customArrow': customArrow, ...rest} = props;
   return (
-    <button style={styles.defaultArrowButton}>
-      {props['dir'] < 0 ? '<<' : '>>'}
+    <Arrow
+      by={-1}
+      customArrow={customArrow || <DefaultArrow by={-1} />}
+      {...rest}
+    />
+  );
+}
+
+/**
+ * @param {!JsonObject} props
+ * @return {PreactDef.Renderable}
+ */
+export function ArrowNext(props) {
+  const {'customArrow': customArrow, ...rest} = props;
+  return (
+    <Arrow
+      by={1}
+      customArrow={customArrow || <DefaultArrow by={1} />}
+      {...rest}
+    />
+  );
+}
+
+/**
+ * @param {!JsonObject} props
+ * @return {PreactDef.Renderable}
+ */
+function DefaultArrow(props) {
+  const {by} = props;
+  const [hover, setHover] = useState(false);
```

Comment:
Hover is always hard to see done in JS. I guess this is our only way for now, but another items for CSS stylesheet. Question however: what does 0.1 do with hover/active? Do we do it exclusively via CSS or also with JS?

Reply:
Hover/active is exclusively done in CSS

Reply:
Is there anything new here that needs JS here? Or is this exclusively the stylesheet problem that we're tracking? If so, please add a `TODO(wg-bento#issue)` so that we can reassess when we have some solutions in that area.

Reply:
This is all styles, added the TODO.